charts/sftpgo/values.yaml

```diff
@@ -163,6 +163,57 @@ services: {}
   #       nodePort:
   #   externalTrafficPolicy:
 
+ingress:
+  # Ingress for the Web UIs (user and admin interface)
+  app:
```

Comment:
Would it make sense to standardize around the name `ui` instead of `app`? It's a little bit inconsistent at the moment.

Reply:
Definitely. I will update the name to be `ui` consistently.